src/main/java/trackitnus/model/task/Task.java

```diff
@@ -15,55 +16,77 @@
  */
 public class Task {
     public static final String TYPE = "T";
-
-    public static final String DATE_MESSAGE_CONSTRAINTS = "Date should be in the format d/MM/yyyy";
+    public static final String DATE_MESSAGE_CONSTRAINTS = "Date should be in the format dd/MM/yyyy or dd/MM/yyyy hh:mm";
     public static final String WEIGHTAGE_MESSAGE_CONSTRAINTS = "Weightage should be in the"
         + " form of a floating point number";
-    public static final String REMARK_MESSAGE_CONSTRAINTS = "Remarks can take any values, and it should not be blank";
-    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("d/MM/yyyy");
+    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+    public static final String VALIDATION_REGEX = "[^\\s].*";
 
     private final Name name;
     private final LocalDate date;
-    private final Address address;
-    private final Double weightage;
+    private final Code code;
     private final String remark;
+    private boolean isComplete;
```

Comment:
LOL It just crossed my mind, I think this is better as isDone (shorter and more "grammatically" correct haha)

Reply:
Hmm ok I shall refactor this